test/integration/registerable_test.rb

```diff
@@ -1,6 +1,8 @@
 require 'test_helper'
 
 class RegistrationTest < Devise::IntegrationTest
+  uses_transaction :test_a_guest_user_should_receive_the_confirmation_instructions_from_the_default_mailer
```

Comment:
Yes, my initial commit doesn't uses the gem for rails 5 :-)  and everything passed.